docs/metrics.md

```diff
@@ -114,6 +114,9 @@ Total VM filesystem capacity in bytes. Type: Gauge.
 ### kubevirt_vmi_filesystem_used_bytes
 Used VM filesystem capacity in bytes. Type: Gauge.
 
+### kubevirt_vmi_last_connection_timestamp_seconds
+Virtual Machine Instance last connection timestamp, including VNC, console, portforward, SSH and usbredir connections. Type: Gauge.
```

Comment:
I think that might be a little misleading. What about connections to VNC/SSH/RDP via a service or secondary interface? As it stands, this would indicate an unused VMI, although it could be accessed via the aforementioned.

Reply:
Updated. Please see https://github.com/kubevirt/kubevirt/pull/11934/files#r1618522817.